src/controllers/phones.ts

```diff
@@ -32,3 +32,10 @@ export const getByPagination = async(req: Request, res: Response) => {
 
   res.send(phones);
 };
+
+export const getRecomendedPhones = async(req: Request, res: Response) => {
+  const { id } = req.params;
+  const phones = await phonesService.getRecomendedPhones(Number(id));
```

Comment:
Why do you convert data to a number in the parameters and then convert it again in the middle of the function?